diff --git a/config/src/font.rs b/config/src/font.rs
index 99b75fcb8..f92ed2969 100644
--- a/config/src/font.rs
+++ b/config/src/font.rs
@@ -431,7 +431,7 @@ impl FontAttributes {
 impl Default for FontAttributes {
     fn default() -> Self {
         Self {
-            family: "JetBrains Mono".into(),
+            family: "monospace".into(),
             weight: FontWeight::default(),
             stretch: FontStretch::default(),
             style: FontStyle::Normal,
